proarc-webapp/src/main/resources/cz/cas/lib/proarc/webapp/client/ClientMessages_cs.properties

```diff
@@ -25,711 +25,675 @@ AboutWindow_Title=O ProArcu
 ## {2} - revision
 AboutWindow_Msg=<div class="proarcAboutWindowMessageTitle">ProArc</div>\
     <p>Verze: <b>{0}</b>\
-    <p>Produk\u010dn\u00ed a archiva\u010dn\u00ed syst\u00e9m je n\u00e1stroj pro v\u00fdrobu a \u00fapravu popisn\u00fdch, \
-    administrativn\u00edch a technick\u00fdch metadat. V\u00fdvoj prob\u00edh\u00e1 pod licenc\u00ed \
+    <p>Produk\u010Dn\u00ED a archiva\u010Dn\u00ED syst\u00E9m je n\u00E1stroj pro v\u00FDrobu a \u00FApravu popisn\u00FDch, \
```

Comment:
Opravené. 

@jkremlacek jak si dosáhl toho, že tvoje lokalizace (které jsi napsal v Idea), jsou lowercase? To jsi psal ručně?

Idea to automaticky převádí na uppercase a NetBeans zase na lowercase :-( https://netbeans.org/bugzilla/show_bug.cgi?id=239513